olm-catalog/serverless-operator/manifests/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Should this be the v1 based CRD?

Reply:
+1 on v1 CRD.

Reply:
I have moved my "old" PR for CRD_v1 now to Ali's brach:

https://github.com/aliok/serverless-operator/pull/5

See the old reference for the size of the DIFF   - I've now closed my old CRD_v1 PR

Reply:
Gonna merge that after CI reports success/fail here

Don't want to interrupt it. I want to see the results

Reply:
Let's do this separately. I had trouble doing this. I received strange OLM errors. Perhaps unrelated.
Doing it in a separate PR makes more sense anyway since we already have the CRD with v1beta1 in master and that change is not in this PR's scope

Reply:
Did my PR for your branch not work @aliok ?

But I can do it later as separate PR 🤷🏻‍♂️

Reply:
Yeah, let's do that separately once we merge this PR.